app/controllers/api/v1/private_controller.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module Api
+  module V1
+    class PrivateController < ApplicationController
+      def sample
```

Comment:
The method name in the controller should follow the Rails Controller method names. We just use the different name in some special case (`download`, ...)

```suggestion
      def index
```